Assets/Python/UniquePowers.py

```diff
@@ -233,17 +206,17 @@ def tamilConquestUP(self, iEnemy, iNumTargets=1):
 
 #------------------ARABIAN U.P.-------------------
 
-	def mughalUP(self, city):
+	def mughalUP(self, city): # Unused
 		return
-		pMughals = gc.getPlayer(iMughals)
+		# pMughals = gc.getPlayer(iMughals)

```

Comment:
I doubt that. The pMughals is also initiated in the Consts. So it should be loaded as well.